koza/model/config/source_config.py

```diff
@@ -68,7 +70,7 @@ class FieldType(str, Enum):
 @dataclass(frozen=True)
 class Filter:
     filter: FilterCode
-    value: Union[str, int, float]
+    value: Union[StrictInt, StrictFloat, List, str]
```

Comment:
Wondering if this should be
```
value: Union[StrictInt, StrictFloat, StrictStr, List[Union[StrictInt, StrictFloat, StrictStr]]
```

We should also raise an exception for using a List for anything but 'in' and vice versa

Reply:
Ooh, yes, definitely.